### PR TITLE
Add Kindle for Mac

### PR DESCRIPTION
**Pull request by naoyes**

### CODE REVIEW

File: Casks/kindle.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'kindle' do
+  version '40648'
+  sha256 'c9549380f777b7c6e2e7c3c1d0c81442c242733ca4879faf543219a125d35773'
+
+  url 'http://kindleformac.amazon.com/#{version}/KindleForMac.dmg'
```
Comment (githubutilities) [inline Casks/kindle.rb]: I think you should use double quote for URL.

**Comment by rolandwalker** [comment]
Hi @naoyes !  Thanks!

Nice to see this Cask coming back – we removed it in #6710 due to changes by Amazon, which appear to have been reversed.

For the `homepage` stanza, it looks like you can use https://www.amazon.com/gp/digital/fiona/kcp-landing-page .

@githubutilities your suggestion about the URL quoting is correct, though it would be kind of you to allow one of the maintainers to respond first before chiming in.

**Comment by naoyes** [comment]
@githubutilities Thank you very much for your suggestion;) I fixed that.

@rolandwalker Hi! This is my first PR in my life.Please let me know if you have any problems.

**Comment by githubutilities** [comment]
@rolandwalker Sorry about the rush, I didn't know about this rule.

**Comment by radeksimko** [comment]
@githubutilities I'm not sure I'd call it a rule, but IMO the whole point is that maintainer may sometimes spot more issues in the code (as maintainers tend to have better knowledge of all stanzas, common issues etc.) and therefore reduce the amount of iterations (modifications) needed from the original contributor.

In other words, it's about not wasting contributor's time/effort as we value all our contributors. :smiley: 

@rolandwalker correct me if I'm wrong

**Comment by rolandwalker** [comment]
Merging this much appreciated first PR!

**Pull request merged**